api/swagger/forms.yaml

```diff
@@ -40,9 +38,10 @@ paths:
       operationId: getForm
       tags:
         - Get Form Definition
-      description: Gets the definition of the given form
+      description: Gets the definition of the given form. 
       parameters:
         - "$ref": "#/components/parameters/formId"
+        - "$ref": "#/components/parameters/acceptLang"
```

Comment:
caching may be concern here. 
we had another end point for retuning locale specific form in which locale was part of path.